.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Could you remove remove your .vscode settings from this PR? Other people maybe have their own settings when building this project.

Reply:
ok

Reply:
I have changed my code.